utils/testgen.py

```diff
@@ -238,11 +237,9 @@ def pytest_generate_tests(metafunc):
         argnames.append('provider')
 
     for provider, data in cfme_data.get('management_systems', {}).iteritems():
-        if data['type'] in cloud_provider_type_map:
-            prov_obj = get_cloud_provider(provider)
-        elif data['type'] in infra_provider_type_map:
-            prov_obj = get_infra_provider(provider)
-        else:
+        try:
```

Comment:
Type checking no longer required, and we just skip, this will skip things like rhos_uc